extensions/amp-analytics/0.1/vendors/webengage.js

```diff
@@ -17,9 +17,12 @@
 import {jsonLiteral} from '../../../../src/json';
 
 const WEBENGAGE_CONFIG = jsonLiteral({
+  'vars': {
+    'clientId': 'CLIENT_ID(we_luid)',
+  },
```

Comment:
Can you implement your previous changes in this file as well?

Reply:
@micajuine-ho I am sorry :( Totally missed it. Done now.